### ISSUE TITLE
Non 48 hour server/Non Advanced Server experiences Apocalypse/Possibly No Chimes Heard By Anyone Interested in Stopping an Apocalypse

### ISSUE THREAD

**Issue by DougLefelhocz** [jasonrohrer/OneLife#1011]
Kortyna posted this image [on the discord](https://cdn.discordapp.com/attachments/328215300279500800/1190793699840491660/image.png?ex=65a3180d&is=6590a30d&hm=d751762c2e71836a7a4df563020182ecbe800f58d586dec3907e0734d391cb15&) from a server1 life.

[Jason Rohrer](https://onehouronelife.com/forums/viewtopic.php?id=4877):

"What if, _at the top of the tech tree_, the most difficult-to-craft item was The Button?"

Of course, "at the top of the tech tree" "the button" doesn't work.

But this happened on server1.  To my knowledge, there was no glass, no radios, no trucks, no oil wells, and no paved roads on server1.  Also, no cameras, I'm guessing.  No advanced towns also.  I did an oil pumpjack on server1 during off hours/only bs2 had incoming players not checking a custom server and someone else who joined me made some more kerosene.  But, I did NOT use it, nor wander back to that town and I believe that doing another oil pumpjack would have made more sense than wandering back to that kerosene, since I did that two or three days ago now.

I also mentioned elsewhere that I had to take down an apocalypse tower earlier in the week.  

Earlier today I had made the argument that Gingers had a valid reason to resettle, since no server1 family has exhausted the game's content this past week, and they could potentially survive in an older town if they got oil up.

About 30 minutes before the apocalypse happened I had visited blacks and tans as [Pippin Harvest](http://lineage.onehouronelife.com/server.php?action=character_page&id=10053786) trying to get rubber supplies, and one had a shallow well, the other just had a ready well site.  It's not like s1 was just advanced towns.

The person who says that s/he did the apocalypse is the same person who expressed anger on the discord at me having taken down the tower this week.  If they were smart about their attempt, they made the key steps when incoming players not checking a custom server were all directed to bigserver2.  I even asked this person "did anyone even hear the two chimes?"  He/she had no response.

"Rare, unsustainable ingredients that you cannot procure while working completely alone."

No, someone can just birth their alt.  In fact, I saw two similar PX names for a bit a few days ago on s1, similar to the person's discord name.  I find it likely that this person did all by their own self. 

"World-wide warnings as the ritual gets closer and closer to completion."

I played as Spoorthi Harvest, Love Harvest, and I think 2 or 3 other lives today in [Harvest family](http://lineage.onehouronelife.com/server.php?action=character_page&id=10052506).  I also played on s1 in [Appler family](http://lineage.onehouronelife.com/server.php?action=character_page&id=8552152&rel_id=8552152). and a few lives in [Hohenwarter](http://lineage.onehouronelife.com/server.php?action=character_page&id=10047358) family before that.  I did not hear a chime in any of those lives, and haven't heard a chime after I took down the previous tower.  Again, I wonder, did anyone **at all** hear the chimes?  Or a second chime?  If no one hears the chime, I don't see how there's any sort of drama.

"Toward the end in a steady state, life is meaningless, and no contribution matters."

I agree on this, and hence generally have either played or tried to help newer families.  But, there is something to getting oil up (AND then improving the process by doing an oil well), doing a paved road, glass, and so on.  There's also something to advanced towns when I played on bs2 a few lives.  I do not agree that this "steady state" had gotten achieved for a single family on server1 when this happened.  Nor the server as a whole.  Again, I observed two early camps.

So, I think we have a non-intended use of the apocalypse here.

Also, as I write this, the arc on bigserver2 has been going on for 917851 OHOL years, some 637 days old.  Before the server merge especially, a very strong case, could get made that an apocalypse should have happened a long time ago, or an arc reset was long overdue.  But, no apocalypse happened (for a variety of reasons).

And instead there's an apocalypse on a server where there's plenty of content that hadn't become relevant.

**Comment by DougLefelhocz** [comment]
Maybe a server needs to have at least 40 players on it for a week... or a month... at all times... before an apocalypse is possible.  Or just no apocalypses at all, since they don't seem needed even when there's a case for their use.

**Comment by DougLefelhocz** [comment]
Later:

"BlueHawk — Today at 8:03 PM

I actually agree. I feel like you shouldn't be able to put the blocks on the tower if the server is below 15 or something, cus it only dings when you go up to the block thats seeping.  So by only being able to put it on while theres 15 or ppl then that means that at least 15 ppl hear the ding."

**Comment by Shadie116** [comment]
AHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH whatever that means.

**Comment by Shadie116** [comment]
Random people have been trying to do the apocalypse and it's never worked because hacks

**Comment by Shadie116** [comment]
besides, it's server .. really thought nobody cared about it. yn because they were all soo advanced with their hacks that show them the status of the entire world

**Comment by cordy-123** [comment]
TL;DR

Players are doing all 4 levels of the endtower alone by themselves in server1 and triggering it immediately when the population increases to 15 players (when people are sent to server1 during server splits and during game updates).

Recommendation:

Change ALL levels of the endtower can only be made when there is at least 15 players in the server.
(The apocalypse itself ALREADY needs 15 players to trigger. https://github.com/jasonrohrer/OneLife/commit/08774f687bd9b9218c78f8b80405672cea2e01a0)

Change ALL levels of the endtower and the apocalypse itself to have the tag "+nearPop_50_10000".

**Comment by cordy-123** [comment]
This is a problem that hits server1 particularly.

The problem is that:

1. you CAN put up the endblocks into the endtower EVEN if there are LESS than 15 people in the server.

(However you can trigger the actual apocalypse using the bloody knife ONLY with 15 people in the server. https://github.com/jasonrohrer/OneLife/commit/08774f687bd9b9218c78f8b80405672cea2e01a0)

2. when Mr. Rohrer changed it so that you have to be close to most of the server population to make an endtower https://github.com/jasonrohrer/OneLifeData7/commit/1b2ff8ddbd7b799204adf9bc7e23227495a9a4ac in response to Bluehawk and others moving hundreds of thousands of tiles east https://github.com/jasonrohrer/OneLife/issues/905, he only changed it for the 1st layer of the endtower

https://onetech.info/2477-Endtower-Base-nearPop_50_10000 
1st level of the endtower - has the tag "+nearPop_50_10000"

The other layers of the endtower AND the actual apoc itself -> DOESN'T have the tag "+nearPop_50_10000"

-----------

server1 in particular is the most affected by these issues.

Since the endtower only does a "monumentCall" when the tower seeps at level 2 ,3, and 4, it is WAY TOO EASY for apoc players to do the apocalypse on server1.

As no one can actually hear the "monumentCall" since they do the endtower alone.

-----------

Currently, this is how they do it.

They set up the endtower all the way to level 4 alone in the server so no one actually hears the monumentCall.

And then when the population goes up, they do the apoc immediately. 

This is either thru game update (when people are shunted off the s1) or server splits. Both of which are VERY EASY to figure out when it happens.

-----------

This does not feel very "sporting" as the whole idea of the apoc is a battle between factions of the server to serve as a story / experience.

Having one person setup all 4 levels of the apoc in secret alone by themselves when no one else is in the server and then, the moment the population goes up, run to their tower (with their alt) to finish up the apoc is not really in-line with that.


-----

**Comment by Shadie116** [comment]
Why is there this long campaign to rewrite every one of Shakespear's plays in the github issue list. I'm sure this c++ software engineer who built an engine inside of a game engine using sprites he drew on paper and set up using his personal gui he built can understand basic sentences. Like damn, yall tryna write the first chapter of the bible???

**Comment by Shadie116** [comment]
Also what factions, they don't got factions anymore. Those don't exist, they got disbanded. It's just a pattern of families that live and die now. Like it was in the beginning, and as it might be.

**Comment by Shadie116** [comment]
Something I do agree on is that solo apocalypse attempts are not really too sporting, honestly. But I don't think they really got a competition considering. *COUGH* 
![image](https://github.com/jasonrohrer/OneLife/assets/126846097/bd223a49-c7c9-4cdd-b9c5-3489d71e8d98)
*ALDPWLH COUGHH*

**Comment by Shadie116** [comment]
one more thing
![image](https://github.com/jasonrohrer/OneLife/assets/126846097/9f55ec6e-56f7-495d-b2ca-872758cb3e9b)

**Comment by DougLefelhocz** [comment]
Bluehawk on January 4, 2024 in game-pictures:

"Did the same thing  before too. ... a couple months ago during an update I apoc'd it."

**Comment by Shadie116** [comment]
bluehawk is snitching on the group chat now  😭 😭 should have just snitched on me instead

**Comment by DougLefelhocz** [comment]
Forman [reported](http://onehouronelife.com/forums/viewtopic.php?id=10842) this also.

**Comment by Shadie116** [comment]
Can you blow my whistle baby, whistle baby. 
![image](https://github.com/jasonrohrer/OneLife/assets/126846097/7ad4af9f-fa31-455a-9700-e298635abe9d)

**Comment by jasonrohrer** [comment]
Ah, nice find, folks!  Will fix.
